src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -188,5 +188,7 @@
   .pf-c-button {
     --pf-c-button--PaddingRight: var(--pf-c-text-input-group__utilities--c-button--PaddingRight);
     --pf-c-button--PaddingLeft: var(--pf-c-text-input-group__utilities--c-button--PaddingLeft);
+
+    display: flex;
```

Comment:
Why do you need this? It throws off the button size - it should fill the height of the component.

<img width="1234" alt="Screen Shot 2022-03-28 at 5 46 01 PM" src="https://user-images.githubusercontent.com/35148959/160499320-ab30333f-7055-42ae-b275-4f6b09af77ff.png">

<img width="1100" alt="Screen Shot 2022-03-28 at 5 46 08 PM" src="https://user-images.githubusercontent.com/35148959/160499332-53f6b19d-7fb9-4b1f-98ea-642763232575.png">

Reply:
The icon was more center-aligned with display: flex.  There is a very slight difference, maybe 1px worth. If you toggle the property on and off using the PR surge link I think you'll notice the same. Can you confirm?